src/debugger/sourceMap.ts

```diff
@@ -67,7 +69,11 @@ export class SourceMapUtil {
 
             if (sourceMap.sources) {
                 sourceMap.sources = sourceMap.sources.map(sourcePath => {
-                    return this.updateSourceMapPath(sourcePath, sourcesRootPath);
+                    if (IS_REMOTE.test(sourcePath) || path.isAbsolute(sourcePath)) {
```

Comment:
I think this might be slightly simplified - you don't need to check if path is absolute ad feed such paths to `updateSourceMapPath` because `path.resolve` recognizes this. In general this might look like
```
return IS_REMOTE(source) ? source : updateSourceMapPath(source)
```

see also https://github.com/Microsoft/vscode-react-native/commit/954784f343d73ace8fbffd7709e71781edecbdeb#diff-2f0685a608d83df323d5af2332f36d95 for example